integrations/evaa_integration.py

```diff
@@ -0,0 +1,107 @@
+from typing import Dict, List, Optional, Tuple
+
+from constants.evaa import EVAA_POOLS_MAP, EVAA_ENDPOINT, EVAA_USDE_START_BLOCK
+
+from integrations.l2_delegation_integration import L2DelegationIntegration
+from integrations.integration_ids import IntegrationID
+from utils.web3_utils import get_block_date
+from utils.request_utils import requests_retry_session
+from utils.slack import slack_message
+
```

Comment:
We're missing some imports here: 

```suggestion
from constants.summary_columns import SummaryColumn
from constants.chains import Chain
```